doc/farm.md

```diff
@@ -81,55 +99,52 @@ farm(exec,
     if (file) return word;
```

Comment:
In line 96 file variable should be passed by value or reference to the lambda function.